astropy/convolution/convolve.py

```diff
@@ -869,3 +870,38 @@ def convolve_models(model, kernel, mode='convolve_fft', **kwargs):
         raise ValueError(f'Mode {mode} is not supported.')
 
     return CompoundModel(mode, model, kernel)
+
+
+def convolve_models_fft(model, kernel, bounding_box, resolution, cache=True, **kwargs):
+    """
+    Convolve two models using `~astropy.convolution.convolve_fft`.
+
+    Parameters
+    ----------
+    model : `~astropy.modeling.core.Model`
+        Functional model
+    kernel : `~astropy.modeling.core.Model`
+        Convolution kernel
+    bounding_box: tuple
+        The bounding box which encompasses enough of the support of both
+        the `model` and `kernel` so that an accurate convolution can be
```

Comment:
```suggestion
        the ``model`` and ``kernel'` so that an accurate convolution can be
```

Reply:
Ithink this is causing the failure in RTD

Reply:
👍